python/Dynamic_Scripts/README.md

```diff
@@ -0,0 +1,5 @@
+# Dynamic Scripts
+
+These scripts perform more complex operations.
+
+The purpose is to explore SecureX Threat Analytics APIs is more details.
```

Comment:
The README needs to contain more elements similar to the base README. Instructions and what not required.